### DISCUSSION TITLE
Zip file flagged as Wacatac.B!.ml in Windows Defender

### DISCUSSION THREAD

**Discussion by h-c-c** [shaise/FreeCAD_SheetMetal#273] [General]
Hi,
FYI this git repo as downloaded directly from here in zip format as well as through the Addon Manager in Freecad is detected as Wacatac.B!.ml on my machine.  Not sure if this is legit, but thought you should know.

Thanks for the Sheetmetal addon.  Can't wait to use it.

Cheers

**Comment by luzpaz** [comment]
See https://github.com/shaise/FreeCAD_SheetMetal/issues/271
